lib/freight/apt.sh

```diff
@@ -78,6 +78,22 @@ apt_filesize() {
     stat -c%s "$(readlink -f "$1")"
 }
 
+# Compute the hash of type $1 for source file $2. If the file has a
+# corresponding `.$filename-$method` file the hash in this file will be returned
+# instead. The hash will alse be recomputed if the source file is newer than the
```

Comment:
s/alse/also

Reply:
Thanks, fixed now. Plus replaced a broken test that shellcheck found, `-nt` isn't supported in sh.